renderer/index.js

```diff
@@ -40,6 +40,7 @@ var dialog = remote.require('dialog')
 
 // For easy debugging in Developer Tools
 var state = global.state = require('./state')
+global.debugLog = require('debug')
```

Comment:
We don't depend on `debug` anymore in webtorrent-desktop, and this doesn't look like it's used in this PR. Will remove.